src/averageAges.js

```diff
@@ -37,7 +41,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenCheck = people.filter(person =>
+    withChildren
+      ? person.sex === 'f' && people.find(child => person.name === child.mother)
+      : person.sex === 'f'
```

Comment:
'f' repeated twice
if that person is someone's mom, it's definitely a woman

Reply:
Good comment, let's fix it